src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  --main-color: #00acdc;
```

Comment:
Better to add it to :root element
https://developer.mozilla.org/en-US/docs/Web/CSS/:root